### PR TITLE
jormun: upgrade to the new (but patched) version of flask-restful

### PR DESCRIPTION
**Pull request by kinnou02**
First step for migrating to python 3: tyr need to be migrated too.

There is some hack in `compat.py`, it will be nice to use the new error format of flask-restful, but it can break some integration.

### CODE REVIEW

**Comment by kinnou02** [comment]
Jenkins, test this please.

**Comment by kinnou02** [comment]
Jenkins, test this please.

File: source/jormungandr/requirements.txt [hunk 1]
```diff
@@ -30,5 +28,7 @@ aniso8601==0.82
 elasticsearch==1.4.0
 ujson==1.33
 urllib3==1.10.1
-numpy==1.9
+numpy==1.9.0
 pybreaker==0.2.3
+amqp==1.4.9
+anyjson==0.3.3
```
Comment (xlqian) [inline source/jormungandr/requirements.txt:33]: amqp and anyjson are newly used in the project?

File: source/jormungandr/jormungandr/exceptions.py [hunk 2]
```diff
@@ -44,9 +44,10 @@ def format_error(code, message):
     return error
 
 
-class RegionNotFound(Exception):
+class RegionNotFound(HTTPException):
```
Comment (xlqian) [inline source/jormungandr/jormungandr/exceptions.py:47]: nice

**Comment by kinnou02** [comment]
require: https://github.com/CanalTP/artemis_references/pull/16 and https://github.com/CanalTP/fabric_navitia/pull/91

**Pull request merged**